src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaConversationPersistenceAdapter.kt

```diff
@@ -41,6 +43,18 @@ class JpaConversationPersistenceAdapter(
       conversationEntityMapper.fromEntityToDomain(it)
     }
 
+  @Transactional(readOnly = true)
+  override fun getPagedConversations(
+    prisonNumber: String,
+    pageNumber: Int?,
+    pageSize: Int?,
+  ): PagedResult<Conversation> {
+    val pageRequest = PageRequest.of(pageNumber ?: 20, pageSize ?: 20)
```

Comment:
Is this right? If you don't specify a pageNumber then you'll get page 20 by default?
Or have I read that wrong?

Reply:
Ah yeah, that would be a mistake. Both should always provided by the `ConversationService` anyway so I'll remove those defaults. 👍